python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -42,13 +41,12 @@ def _copy_or_convert_data(
 ):
     """Copy or convert the data from input_path to output_path."""
     os.makedirs(os.path.dirname(output_path), exist_ok=True)
-    if input_format == "numpy":
-        # If the original format is numpy, just copy the file.
-        shutil.copyfile(input_path, output_path)
-    else:
-        # If the original format is not numpy, convert it to numpy.
-        data = read_data(input_path, input_format, in_memory)
-        save_data(data, output_path, output_format)
+    # Read data. If the original format is not numpy, convert it to numpy. If
+    # dim of the data is 1, reshape it to n * 1 and save it to output_path.
+    data = read_data(input_path, input_format, in_memory)
```

Comment:
blindly read and save may incur performance issue for large file which is in expected dims such ogb-lsc-mag240m dataset whose feature data is in several hundred GB. please take this into consideration.